newIDE/app/src/InstancesEditor/HighlightedInstance.js

```diff
@@ -81,13 +81,14 @@ export default class InstancesSelection {
       highlightedInstance.getObjectName() +
       '\n' +
       'X: ' +
-      parseInt(highlightedInstance.getX()) +
+      Math.round(highlightedInstance.getX() * 100) / 100 + // An instance position can have a lot of decimals, so round to 2 decimals.
```

Comment:
Nit: this could be a roundUpTo(2, highlightedInstance.getX()) method, as this could allow in the future to easily adapt the rounding to 0, 1 or 2 decimal places.

Reply:
I'm under the impression that this tooltip will disappear, or at least the content will change, as it doesn't seem to be used for these information.
So for the moment, not touching this